homeassistant/components/neurio_energy/sensor.py

```diff
@@ -54,28 +69,81 @@ def update_active():
         """Update the active power usage."""
         data.get_active_power()
 
+    @Throttle(MIN_TIME_BETWEEN_ACTIVE_UPDATES)
+    def update_generation():
+        """Update the active power usage."""
+        data.get_active_generation()
+
+    @Throttle(MIN_TIME_BETWEEN_DAILY_UPDATES)
+    def update_generation_daily():
+        """Update the active power usage."""
+        data.get_daily_generation()
+
+    @Throttle(MIN_TIME_BETWEEN_DAILY_UPDATES)
+    def update_net_consumption():
+        """Update the active power usage."""
+        data.get_net_consumption()
+
+    update_dataset()
     update_daily()
     update_active()
-
+    update_generation()
+    update_generation_daily()
+    update_net_consumption()
+
+    # Update Dataset
+    add_entities([NeurioEnergy(data,
+                               JSON_DATASET,
+                               DAILY_TYPE,
+                               update_dataset)]
+                 )
     # Active power sensor
-    add_entities([NeurioEnergy(data, ACTIVE_NAME, ACTIVE_TYPE, update_active)])
+    add_entities([NeurioEnergy(data,
```

Comment:
Please revert all the style changes to this file. They make it impossible to see what has changed.

Reply:
What?

Reply:
I had to do this to get the code to pass the validation. Otherwise I kept getting errors for the lines being too long.